relay/circuit-v2.md

```diff
@@ -309,11 +345,13 @@ The relay sends a `StopMessage` with `type = CONNECT` and the following form:
 ```
 StopMessage {
   type = CONNECT
-  peer = Peer { ID = ...}
+  initiator = Peer { ID = ...}
+  target = Peer { addrs = ...}
```

Comment:
I don't quite like the name `target` here, perhaps use `destination` or just `dest`.

Reply:
also, we can call `initiator` as `src` or `source` to match standard nomenclature.